tests/test_env/test_env_cashpenalty.py

```diff
@@ -67,6 +67,23 @@ def test_shares_increment(self):
         self.assertEqual(holdings[0], 10.0)
         self.assertEqual(holdings[1], 0.0)
 
+    def test_patient(self):
```

Comment:
can you please add test for the opposite case ? When `patient=False` then is_done==True`?